example/ts/exampleApi.ts

```diff
@@ -366,7 +366,7 @@ class ExampleApiHttpClient implements IExampleApi {
 		return fetchResponse(this._fetch, this._baseUri + uri, fetchRequest)
 			.then(result => {
 				const status = result.response.status;
-				let value: IGetWidgetsResponse = null;
+				let value: IGetWidgetsResponse | null = null;
```

Comment:
Can/should this be `IGetWidgetsResponse?` instead?

Reply:
It cannot.

Reply:
Why not?

Reply:
Because it's not valid TypeScript. ;)

Reply:
Oh! 😊